### DISCUSSION TITLE
Add a menu with a markdown guide

### DISCUSSION THREAD

**Discussion by octokatherine** [katherinepeterson/readme.so#154] [Ideas]
Add a markdown guide for reference that shows the syntax for all basic markdown components.

**Comment by saurabhchaturvedy** [comment]
@katherinepeterson in the menu do we want create a simplified guide of our own from scratch ?  Another good idea could be, once user clicks the menu he gets redirected to  https://www.markdownguide.org/  in a separate tab?  This is like an opensource website guiding standards for markdown creation .

**Comment by kayijyc** [comment]
@katherinepeterson Is this feature request still open? I would also suggest something like dev.to's markdown guide which would be a modal that consists of basic markdown syntax. Below shows an example of what I mean 😉 

<img width="1380" alt="Screenshot 2021-04-28 at 08 57 41" src="https://user-images.githubusercontent.com/45145081/116360489-440b7500-a800-11eb-9fc4-27211b4b1660.png">

A bonus feature that I can think of is for the user to be able to click on the markdown component that they want and the syntax would be copied to clipboard. The CTA to open the modal could be placed next to the download button on the navbar.

What do you think? If you think this is good, I'd be happy to take this up 🙂

**Reply by octokatherine** [reply]
@annacjy sorry I missed your comment, but yes that is exactly like what I was thinking

**Reply by kayijyc** [reply]
@octokatherine it's all good. i think this is a great additional feature that will be beneficial. 

i'd still be available to take this up, but since this is no longer an issue but a discussion, please let me know how would you like to proceed 🙂

**Reply by octokatherine** [reply]
@annacjy I'll make an issue!

**Reply by chnuessli** [reply]
markdownguide.org is a good reference. It's open source too. Maybe you both can set up interaction?
